data-reconciliation-app/test/e2e-test/src/data.ts

```diff
@@ -0,0 +1,50 @@
+export const member0DataPart1 = [
```

Comment:
Would these be better as flat files so if Brent gives us a data set we could easily test them?

Reply:
I think implementing #131 will help solving this point. I have created a follow up issue: #143